### ISSUE TITLE
Adding a map system

### ISSUE THREAD

**Issue by devnullone** [devnullone/didobin#3]
Consider choosing one based on Openstreetmap like Leaflet, Mapbox etc...

**Comment by Shamik1836** [comment]
What type of location you want to show here, I can work with mapbox

**Comment by ansharora28** [comment]
I can do this :)

**Comment by devnullone** [comment]
> What type of location you want to show here, I can work with mapbox

@Shamik1836  Please consider look on this https://github.com/mapbox/mapbox-gl-markers

**Comment by Shamik1836** [comment]
Yeah it's showing a interactive map on North Atlantic Ocean. Can I add this only or some other location?

**Comment by Shamik1836** [comment]
![image](https://user-images.githubusercontent.com/83940432/194001673-13e40ae6-cdb9-4dce-858e-d1001b8e186b.png)
It's howing like this should I add this to the repo?

**Comment by devnullone** [comment]
> Yeah it's showing a interactive map on North Atlantic Ocean. Can I add this only or some other location?
It yours!
You can got with one marker and make a PR if done. I will update after according the real geolocation!

**Comment by devnullone** [comment]
> As I see, there are 5 pages on the site namely booking,contacts,howto,root,mission,pricing. On which page I need to add the map @devnullone

contact one as well !

**Comment by Aniketmishra0** [comment]
Hello Sir,
Can You assign this issue to me?

**Comment by devnullone** [comment]
> Hello Sir, Can You assign this issue to me?

Hi @Aniketmishra0, Thank for your interest. 
Please shoot a pr first!
